src/SearchQuery.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace DigitalBackstage\Searchable;
+
+interface SearchQuery
+{
+    /**
+     * An associative array <field> => ['value' => 42, 'type' => 'integer', ...]
+     */
+    public function expression();
```

Comment:
What if you need nested conditions? or that the current condition level needs to be OR-cased.
And custom types, like handling ValueObjects. Etc.

This some more though and standardization 😃 I know it's complex.

Reply:
Yeah this is a bit naive at the moment, should probably be a tree. Maybe I just need an `Expression` interface to abstract this away, and a very simple implementation that does not need several levels at first?